Mathlib/Data/Complex/Abs.lean

```diff
@@ -348,3 +348,13 @@ theorem lim_abs (f : CauSeq ℂ Complex.abs) : lim (cauSeqAbs f) = Complex.abs (
     let ⟨i, hi⟩ := equiv_lim f ε ε0
     ⟨i, fun j hj => lt_of_le_of_lt (Complex.abs.abs_abv_sub_le_abv_sub _ _) (hi j hj)⟩
 #align complex.lim_abs Complex.lim_abs
+
+/-!
+### Some useful lemmas
+-/
```

Comment:
```suggestion
```
I don't think we need to add this comment, mathlib is literally full of small useful lemma.